components/content/List/List.css

```diff
@@ -0,0 +1,70 @@
+diamond-list {
+  --diamond-list-spacing: 0;
+
+  li {
+    margin-bottom: var(--diamond-list-spacing);
```

Comment:
Should we be using margin-block-* and margin-inline-* in here because it involves typography? Maybe it only matters for the inline spacing on the icons, do any languages have horizontal lists 😵

Reply:
I'll leave type direction support for a whole 'nother PR 😅